README.md

```diff
@@ -357,6 +333,94 @@ To get the hostname of the master you can run:
 It's unfortunate that you have no flexibility here. This restriction which will hopefully
 be lifted in the future.
 
+</details>
+
+# <a name="running-tests"></a> Running Tests
+## <a name="basic-cluster-setup"></a> Basic Cluster Setup
+
+On the coordinator node:
+
+```bash
+# Setup your test cluster with PostgreSQL 11.5 and Citus master branch
+fab use.postgres:11.5 use.citus:master setup.basic_testing
+
+# Lets change some conf values 
+fab pg.set_config:max_wal_size,"'5GB'"
+fab pg.set_config:max_connections,1000
+
+# And restart the cluster
+fab pg.restart
+```
+
+
+## <a name="pgbench"></a> Running PgBench Tests
+
+On the coordinator node:
+
+```bash
+# This will run default pgBench tests with PG=11.5 and Citus Enterprise 9.0 and 8.3 release branches
+# and it will log results to pgbench_results_{timemark}.csv file
+# Yes, that's all :) You can change settings in fabfile/pgbench_confs/pgbench_default.ini
+fab run.pgbench_tests
+
+# It's possible to provide another configuration file for tests
+# Such as with this, we run the same set of default pgBench tests without transactions
+fab run.pgbench_tests:pgbench_default_without_transaction.ini
+```
+
+## <a name="scale"></a> Running Scale Tests
+
+On the coordinator node:
+
+```bash
+# This will run scale tests with PG=11.5 and Citus Enterprise 9.0 and 8.3 release branches
+# and it will log results to pgbench_results_{timemark}.csv file
+# You can change settings in files under the fabfile/pgbench_confs/ directory
+fab run.pgbench_tests:scale_test.ini
+fab run.pgbench_tests:scale_test_no_index.ini
+fab run.pgbench_tests:scale_test_prepared.ini
+fab run.pgbench_tests:scale_test_reference.ini
+fab run.pgbench_tests:scale_test_foreign.ini
+fab run.pgbench_tests:scale_test_100_columns.ini
+```
+
+## <a name="pgbench-cloud"></a> Running PgBench Tests Against Citus Cloud
```

Comment:
I think we should not refer to `CitusCloud` anymore. Can you replace all references to `Hyperscale (Citus)` ?